docs/feature-flags/audit-logs.mdx

```diff
@@ -0,0 +1,630 @@
+---
+title: Audit Logs
+# sidebar_position:
+slug: /feature-flags/audit-logs
+description: Use the flag triggers to automatically turn feature flags on or off by calling a URL using the POST method.
+tags: ['log', 'audit-logs']
+---
+
+The Audit Logs feature stores all log history from the current Environment. You can use the logs to identify problems, or who performed modification in your project flags. 
+
+To access the logs associated with the current environment, select **Audit Logs** on the dashboard sidebar. To check logs from other environments, you first need to change to the desired environment and then access the Audit Logs page.
```

Comment:
We could simplify this.

`To access the logs associated with the current environment, select Audit Logs on the dashboard sidebar.` -> `To check the logs, access the Audit Logs page from the sidebar menu.`